iothub/device/samples/PnpDeviceSamples/TemperatureController/TemperatureController.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootDir>$(MSBuildProjectDirectory)\..\..\..\..\..</RootDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
+    <!--<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.28.0" /> -->
+    <!--<PackageReference Include="Microsoft.Azure.Devices.Provisioning.Client" Version="1.6.0" /> -->
+    <!--<PackageReference Include="Microsoft.Azure.Devices.Provisioning.Transport.Mqtt" Version="1.3.0" /> -->
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.8" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\..\provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj" />
```

Comment:
This package reference will need to be updated to nuget reference - once the private nugets are published